### PR TITLE
Add Web Frameworks.

### PR DESCRIPTION
**Pull request by nirgn975**
Edit the [onion](https://github.com/davidmoreno/onion) place from `network` to `web frameworks` because author describe it as `C library to create simple HTTP servers and Web Applications.`

And add 2 more: [kore](https://github.com/jorisvink/kore) and [klone](https://github.com/koanlogic/klone).

### CODE REVIEW

**Comment by nirgn975** [comment]
@aleksandar-todorovic Ready for review.

**Comment by nirgn975** [comment]
@aleksandar-todorovic Is there something I can do for you yo merge this PR ?

**Comment by aleksandar-todorovic** [comment]
Whoops, sorry, I didn't notice the notification. Looking at your changes now.

File: README.md [hunk 1]
```diff
@@ -897,7 +895,6 @@ This is a 'catch-all' category for anything that doesn't fit well anywhere else.
 [358]: http://www.samnip.ps/thought/macro-storage-for-inverse-comma
 [359]: https://github.com/awslabs/s2n
 [360]: https://www.gnu.org/software/recutils/
-[361]: http://pp.ipd.kit.edu/firm/Index
```
Comment (aleksandar-todorovic) [inline README.md]: Seems like this is up, maybe it was just temporarily down?

**Comment by nirgn975** [comment]
@aleksandar-todorovic Yes, looks like it was just temporarily, I return it.

**Comment by aleksandar-todorovic** [comment]
Thanks!

I'll fix this broken license thing in a separate commit.

**Pull request merged**